spec/macros/resource_macros.rb

```diff
@@ -41,6 +41,7 @@ def it_should_be_creatable(options = {})
 
         VCR.use_cassette("#{described_class.to_s}_create_delete") do
           @creatable_object.destroy
+          sleep(1)
```

Comment:
A consideration, maybe not even for this PR:

if the object gets returned even after deletion, maybe it has an attribute `deleted: true` or something we can check and make the test pass?